parsec/backend/postgresql/init_tables.sql

```diff
@@ -28,6 +28,12 @@ CREATE TABLE user_ (
     -- NULL if certifier is the Root Verify Key
     user_certifier INTEGER,
     created_on TIMESTAMPTZ NOT NULL,
+    -- NULL if not yet revoked
+    revoked_on TIMESTAMPTZ,
```

Comment:
So `TIMESTAMPTZ` works in with postgresql but not sqlite?

Reply:
yep, see https://www.postgresql.org/docs/10/datatype-datetime.html:
> Note
> The SQL standard requires that writing just timestamp be equivalent to timestamp without time zone, and PostgreSQL honors that behavior. **timestamptz** is accepted as an abbreviation for timestamp with time zone; this is a PostgreSQL extension.